sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -1405,24 +1398,24 @@ public void populateDisplayData(DisplayData.Builder builder) {
       public static class Configuration {
 
         // All byte arrays are UTF-8 encoded strings
-        private Iterable<KV<byte[], byte[]>> producerConfig;
-        private byte[] topic;
-        private byte[] keySerializer;
-        private byte[] valueSerializer;
+        private Iterable<KV<String, String>> producerConfig;
+        private String topic;
+        private String keySerializer;
+        private String valueSerializer;
```

Comment:
If you change the types here this will affect the lookup of the configuration fields in ExpansionService. The lookup is performed based on the returned type of the coder. The test still uses the bytes coder, so this will attempt to lookup a `byte[]` field.

Reply:
This is for the KafkaIOExternal test which I see failing.